README.md

```diff
@@ -234,35 +238,35 @@ Instructs ESLint to try to fix as many issues as possible, see https://eslint.or
 The following steps describe the minimal steps required to successfully log in to the portal UI.
 
 1. Set up Docker, as described above: [Docker](README.md#docker)
-2. Set up your `.env` file per [Environment Variables](README.md#environment-variables)
-3. Ensure your `SSP_KK_SECRET` matches the value in the Keycloak admin console. If it does not, please try [Reset your keycloak realm](./KeycloakAdvancedSetup.md#reset-realm). If all else fails, manually generate your own secret by following: [Generating SSP_KK_SECRET](./KeycloakAdvancedSetup.md#generating-ssp_kk_secret)
-4. Run the following to install dependencies:
+1. Set up your `.env` file per [Environment Variables](README.md#environment-variables)
+1. Run the following to install dependencies:
    ```
    npm install
    ```
-5. Run the following to start the API and React front-end:
+1. Run the following to start the API and React front-end:
    ```
    npm run dev
    ```
    Successfully running this will result in the self-serve-portal opening in the browser.
-6. Run the following to build the database schema:
+1. Run the following to build the database schema:
    ```
    npm run knex:migrate:latest
    ```
-7. Run the following to populate test data:
+1. Run the following to populate test data:
    ```
    npm run knex:seed:run
    ```
-8. Run the Admin service locally by following [Connecting to local Admin service](#connecting-to-local-admin-service)
-9. Create an account in the UI by clicking `Create Account`. You can use a fake email address since we use [MailHog](https://github.com/mailhog/MailHog) to capture emails and store them locally.
-10. Go to local MailHog at http://localhost:18025/ and you will see an email from `test@self-serve-portal.com` with the subject `Verify email`
-11. Open the email and Click `Verify Email`
-12. Fill in the form however you want and submit the form
-13. Connect to the database server `localhost,11433` using the credentials in [docker-compose.yml](docker-compose.yml) under `KC_DB_USERNAME` and `KC_DB_PASSWORD`
-14. In the `uid2_selfserve` database, observe that `dbo.users` now contains a row with with the details you just filled out.
-15. Approve your account by updating the `status` of the row in `dbo.participants` that corresponds to your new user, i.e.
+1. Create an account in the UI by clicking `Request Account`. You can use a fake email address since we use [MailHog](https://github.com/mailhog/MailHog) to capture emails and store them locally. Regardless, using the [example.com](https://en.wikipedia.org/wiki/Example.com) domain is encouraged.
+1. Go to local MailHog at http://localhost:18025/ and you will see an email from `noreply@unifiedid.com` with the subject `Verify email`
+1. Open the email and Click `Verify Email`
+1. Fill in the form however you want and submit the form
```

Comment:
Actually I think here for example, since these three steps for MailHog are related, they should be numbered

Reply:
Per my above comment, I am preserving the numbering. Unless you are referring to something else?